cdk/domino_cdk/config_template.yaml

```diff
@@ -40,6 +40,7 @@ route53:
 eks:
   version: "1.19"
   private_api: false
+  secrets_encryption_key_arn: arn:aws:kms:us-west-2:890728157128:key/b9c778f3-46cc-4416-8867-4ab47d912bf6
```

Comment:
This is one of our keys in AWS KMS. If we go forward with making this field mandatory, we need to put some key ARN by default. There is no difference which key to use.